app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: 'from@example.com'
```

Comment:
Use ` default from: Rails.configuration.sender_email` as default and remove `default` from `UserMailer`